src/app/lib/layoutGrid.js

```diff
@@ -18,8 +18,8 @@ import {
 const group4ColWidth = `6.75rem`;
 /* (1008px - (2*16px margins + 7*16px gutters) / 8 columns = 108px = 6.75rem single column width */
 
-const group5ColWidth = `6.9rem`;
-/* (1280px - (2*16px margins + 9*16px gutters)  / 10 columns = 110.4px = 6.9rem single column width */
+const group5ColWidth = `2.95rem`;
+/* (1280px - (2*16px margins + 19*16px gutters)  / 20 columns = 110.4px = 2.95rem single column width */
```

Comment:
```suggestion
/* (1280px - (2*16px margins + 19*16px gutters)  / 20 columns = 47.2px = 2.95rem single column width */
```

Margins&gutters `2*16+19*16 = 336`
Max-width - Margins&gutters = `1280-336 = 944`
Max-width - Margins&gutters / 20-columns = `(1280-336)/20 = 47.2`